### ISSUE TITLE
Can't install dive 

### ISSUE THREAD

**Issue by aaryankb** [ffaanngg/dive#2]
![image](https://user-images.githubusercontent.com/81513586/173230585-d882bde6-e915-42c1-af49-e116e6868d28.png)

This is what I get when I run `dive.exe`

**Comment by shhivv** [comment]
You need to mention the regex pattern for the file you are searching for as the first argument.

**Comment by aaryankb** [comment]
I did not search anything . The error is what I get when I run `dive.exe`

**Comment by shhivv** [comment]
Yes, you need to search for something to use dive. If you are looking for a help prompt, I am sorry dive does not have that at the moment. Feel free to open a PR for the help message tho :)
